src/Pim/index.tsx

```diff
@@ -395,6 +395,25 @@ class Pim extends React.PureComponent {
     const { collectionsAddressBook, collectionsCalendar, collectionsTaskList, addressBookItems, calendarItems, taskListItems } = itemsSelector(this.props);
     const [contacts, events, tasks] = itemValuesSelector(this.props);
 
+    const birthdays = contacts.filter((c) => c.bday).map((c) => {
+      let { year, month, day } = c.bday._time;
+      year = year || '2020';
```

Comment:
Having the default year as 2020 feels wrong. It should probably just be a recurring event starting from the person's date of birth and if not, start from 1900.